### PR TITLE
Feature/update usage examples

### PR DESCRIPTION
**Pull request by vineetpant**
- Added generate keys section to DIDCOMM plugin docs
- Changed vade-evan-cli command tabs in usage-examples
- Updated References

### CODE REVIEW

File: docs/plugins/didcomm.md [hunk 1]
```diff
@@ -32,17 +34,13 @@ _NOTE_: When you send any message that will be encrypted, you need to have a fin
 
 The two functions [`didcomm_send`] and [`didcomm_receive`] can be called with two parameters, `options` and `payload`, the output is an Object of [`VadeDidCommPluginOutput`]:
 
-- Options: [`DidCommOptions`] object,Contains specific information for passing special configuration to the vade_didcomm. Currently its just used to inject specific encryption configuration, to overwrite the default DIDComm DID exchange key encryption and signing.
+- Options: [`DidCommOptions`] object,Contains specific information for passing special configuration to the vade_didcomm. Currently its just used to inject specific encryption configuration, to overwrite the default DIDComm DID exchange key encryption.
```
Comment (wulfraem) [inline docs/plugins/didcomm.md]:
```suggestion
- Options: [`DidCommOptions`] object, Contains specific information for passing special configuration to the vade_didcomm. Currently its just used to inject specific encryption configuration, to overwrite the default DIDComm DID exchange key encryption.
```

File: docs/plugins/didcomm.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 sidebar_position: 3
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # DIDcomm
```
Comment (wulfraem) [inline docs/plugins/didcomm.md]:
```suggestion
# DIDComm
```

"DIDComm" should be the correct ways to write it, saw this also in line 9 and 270, but couldn't make a suggestion in this PR reivew. :D

File: docs/plugins/didcomm.md [hunk 3]
```diff
@@ -162,10 +160,71 @@ The two functions [`didcomm_send`] and [`didcomm_receive`] can be called with tw
 
 The data that is represented in `message` and `metadata` is protocol specific. The message is also attached unencrypted as `messageRaw`.
 
+## Generating keys for didcomm communication
+
+As mentioned above DIDCOMM exchange requires the encryption keys to be passed as [`didcommoptions`], to generate those keys the  `create-keys` subcommand can be used in `vade-evan-cli` for `didcomm` to generate keys for sender and receiver both, once the `sender` and `receiver` keys are generated, they can use those keys to create [`didcommoptions`] for [`didcomm_send`] and [`didcomm_receive`].
```
Comment (wulfraem) [inline docs/plugins/didcomm.md]:
```suggestion
## Generating keys for DIDComm communication

As mentioned above DIDComm exchange requires the encryption keys to be passed as [`DidCommOptions`], to generate those keys the  `create-keys` subcommand can be used in `vade-evan-cli` for `didcomm` to generate keys for sender and receiver both, once the `sender` and `receiver` keys are generated, they can use those keys to create [`DidCommOptions`] for [`didcomm_send`] and [`didcomm_receive`].
```

File: docs/plugins/didcomm.md [hunk 4]
```diff
@@ -374,30 +436,39 @@ payload={
...
 ```sh
 ./vade_evan_cli didcomm receive --options $option --payload $payload
 ```
 
+</TabItem>
+</Tabs>
+
 ## General message types
 
 In some protocols there are extra messages need to be sent to indicate error or successful message exchange to end the session, these messages have following structures:-
```
Comment (wulfraem) [inline docs/plugins/didcomm.md]:
```suggestion
In some protocols there are extra messages need to be sent to indicate error or successful message exchange to end the session, these messages have following structures:
```

File: docs/usage samples/didcomm_issue-credential_holder_request.mdx [hunk 5]
```diff
@@ -91,10 +92,6 @@ payload = {
 <TabItem value="vade_evan_cli" label="vade_evan_cli">
 
 ```sh
-vade_options='{"encryptionKeys":{"encryptionMySecret":"5046adc1dba838867b2bbbfdd0c3423e58b57970b5267a90f57960924a87f156","encryptionOthersPublic":"d92f5eeaa24fd4e66221c770f704a5e2639a476bab82cfec40bd2874abeb481f"}}'
```
Comment (wulfraem) [inline docs/usage samples/didcomm_issue-credential_holder_request.mdx]: (This and the following sections)

Why are the values removed here?

File: docs/usage samples/didcomm_issue-credential_holder_proposes.mdx [hunk 6]
```diff
@@ -39,7 +40,7 @@ If Bob wants a credential issued from Alice , he would need to connect with Alic
 
 The initial communication as per [`issue credential protocol`] would involve following steps :-
 
-Note: Please checkout the documentation for [`didcomm_send`] and [`didcomm_receive`] to understand the `payload` and `options`. The `body` parameter in payload contains the structure required for [`issue credential protocol`] and state represents the current state of protocol as each protocol goes through various status before the communication is completed or terminated. [`issue credential protocol`] starts with proposal sending from Bob so the initial state is `SendProposeCredential`.
+Note: Please checkout the documentation for [`didcomm_send`] and [`didcomm_receive`] to understand the `payload` and `options`. The `body` parameter in payload contains the structure required for [`issue credential protocol`] and state represents the current state of protocol as each protocol goes through various status before the communication is completed or terminated. [`issue credential protocol`] starts with proposal sending from Bob so the initial state is `SendProposeCredential`.
```
Comment (wulfraem) [inline docs/usage samples/didcomm_issue-credential_holder_proposes.mdx]:
```suggestion
Note: Please checkout the documentation for [`didcomm_send`] and [`didcomm_receive`] to understand the `payload` and `options`. The `body` parameter in payload contains the structure required for [`issue credential protocol`] and state represents the current state of protocol as each protocol goes through various status before the communication is completed or terminated. [`issue credential protocol`] starts with proposal sending from Bob so the initial state is `SendProposeCredential`.
```

File: docs/usage samples/didcomm_issue-credential_holder_proposes.mdx [hunk 7]
```diff
@@ -92,10 +93,6 @@ payload = {
 <TabItem value="vade_evan_cli" label="vade_evan_cli">
 
 ```sh
-vade_options='{"encryptionKeys":{"encryptionMySecret":"5046adc1dba838867b2bbbfdd0c3423e58b57970b5267a90f57960924a87f156","encryptionOthersPublic":"d92f5eeaa24fd4e66221c770f704a5e2639a476bab82cfec40bd2874abeb481f"}}'
```
Comment (wulfraem) [inline docs/usage samples/didcomm_issue-credential_holder_proposes.mdx]: (This and the following sections)

Why are the values removed here?

**Review by wulfraem — CHANGES_REQUESTED** [review]

File: docs/usage samples/didcomm_issue-credential_holder_proposes.mdx (hunk 7 above)
Comment (vineetpant) [reply docs/usage samples/didcomm_issue-credential_holder_proposes.mdx]: The values in json args tab are the same values included in the vade_cli tab , i thought it was redundant to have same values there as they have been mentioned in previous tab already ..

File: docs/usage samples/didcomm_issue-credential_holder_request.mdx (hunk 5 above)
Comment (vineetpant) [reply docs/usage samples/didcomm_issue-credential_holder_request.mdx]: The values in json args tab are the same values included in the vade_cli tab , i thought it was redundant to have same values there as they have been mentioned in previous tab already ..

File: docs/usage samples/didcomm_issue-credential_holder_proposes.mdx (hunk 7 above)
Comment (vineetpant) [reply docs/usage samples/didcomm_issue-credential_holder_proposes.mdx]: reverted the changes now

File: docs/usage samples/didcomm_issue-credential_holder_request.mdx (hunk 5 above)
Comment (vineetpant) [reply docs/usage samples/didcomm_issue-credential_holder_request.mdx]: reverted the changes now

**Review by wulfraem — APPROVED** [review]

**Pull request merged**